app/controllers/ems_common.rb

```diff
@@ -834,6 +837,8 @@ def get_form_vars
     @edit[:new][:bearer_token] = params[:bearer_token] if params[:bearer_token]
     @edit[:new][:bearer_verify] = params[:bearer_verify] if params[:bearer_verify]
 
+    @edit[:new][:service_account] = params[:service_account] if params[:service_account]
+
```

Comment:
Since this is an angular form, you should not save the ```params``` data in ```@edit```.
In fact, this method wouldn't even be called on a ```Save``` 
```set_ems_record_vars``` in ```ems_cloud_controller``` is where we save the Cloud Provider form data.